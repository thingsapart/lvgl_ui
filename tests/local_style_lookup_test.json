[
  {
    "type": "label",
    "id": "@test_label_style_lookup",
    "named": "my_style_lookup_label",
    "text": "Direct Prop Test",
    "width": 150,
    "align": "left_mid",
    "radius": 10,
    "scroll_on_focus": false
  }
]
