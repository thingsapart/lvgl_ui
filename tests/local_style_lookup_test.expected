// Code generated by LVGL UI Generator

#include "lvgl.h"

// Forward declaration for any registered C pointers you might have from your application.
// Example: extern const lv_font_t my_custom_font;
void create_ui(lv_obj_t* parent) {
    {
        lv_obj_t* my_style_lookup_label = lv_label_create(parent);
        lv_label_set_text(my_style_lookup_label, "Direct Prop Test");
        lv_obj_set_style_width(my_style_lookup_label, 150);
        lv_obj_set_style_align(my_style_lookup_label, "left_mid");
        lv_obj_set_style_radius(my_style_lookup_label, 10);
        registry_add_pointer(ui_registry, my_style_lookup_label, "test_label_style_lookup", "label");
    }
}
